src/CalcManager/Ratpack/exp.cpp

```diff
@@ -297,7 +297,7 @@ void powrat(PRAT *px, PRAT y, uint32_t radix, int32_t precision)
     }
 }
 
-void powratNumeratorDenominator(PRAT *px, PRAT y, uint32_t radix, int32_t precision)
+void powratNumeratorDenominatorPartial(PRAT *px, PRAT y, uint32_t radix, int32_t precision)
```

Comment:
Let's stay consistent with the rest of ratpak and declare this in ratpak.h

Reply:
Please also add a comment description of the function similar to the one on powrat, describing what calculation it's performing. We should also add one for powratNumeratorDenominator below.

Reply:
I added declaration in ratpak.h and added a comment description as you suggested.